casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -169,23 +169,11 @@ case class TestNode[F[_]: Concurrent: Timer](
                 _   = assert(res.isRight, s"Deploy error ${deploy} with\n ${res.left}")
               } yield ()
           )
-      preState          <- MultiParentCasper.getPreStateForNewBlock[F]
-      creatorsPk        = validatorIdOpt.get.publicKey
-      creatorsId        = ByteString.copyFrom(creatorsPk.bytes)
-      creatorsLatestOpt = preState.justifications.find(_.sender == creatorsId)
-      nextSeqNum        = creatorsLatestOpt.map(_.seqNum + 1).getOrElse(0L)
-      nextBlockNum      = preState.justifications.map(_.blockNum).max + 1
+      preState <- MultiParentCasper.getPreStateForNewBlock[F]
       createBlockResult <- BlockCreator(validatorIdOpt.get, shardName).create(
-                            preState.preStateHash,
-                            preState.justifications.map(_.blockHash),
-                            preState.fringeBondsMap,
+                            preState,
                             preState.rejectedDeploys,
```

Comment:
Here `rejectedDeploys` field is used, while the Proposer uses `fringeRejectedDeploys`. I just want to pay attention. Is this correct?

Reply:
It should be `fringeRejectedDeploys ` as in Proposer, you are spot on.